python/ionpy/Port.py

```diff
@@ -39,3 +54,53 @@ def __getitem__(self, index):
     def __del__(self):
         if self.obj: # check not nullptr
             ion_port_destroy(self.obj)
+
+    
+        # should use numpy type?
+    def bind_i8(self, v: int):
```

Comment:
It is better design to provide a generic `bind` instead of type-specific method (e.g. bind_i8). The appropreate C-API can be determined by inspecting `self.type` and `self.dim`.